### ISSUE TITLE
Weight Decay Fix Original Paper

### ISSUE THREAD

**Issue by PetrochukM** [huggingface/pytorch-pretrained-BERT#188]
Hi There!

Is the weight decay fix from?
https://arxiv.org/abs/1711.05101

Thanks!

**Comment by artemisart** [comment]
Yes
